distributions/standalone/assembly-whitelist.txt

```diff
@@ -154,12 +154,16 @@
 /mule-standalone-${productVersion}/lib/mule/mule-module-sockets-${productVersion}.jar
 /mule-standalone-${productVersion}/lib/mule/mule-module-tls-${productVersion}.jar
 /mule-standalone-${productVersion}/lib/mule/mule-module-tomcat-${productVersion}.jar
-/mule-standalone-${productVersion}/lib/mule/mule-module-validation-${productVersion}.jar
```

Comment:
I looked at this and the only concern that I have is related to a dependency used in both extensions (file and ftp): mule-module-file-extension-common. It is declared as provided in both extensions. Should we keep that or bundle it with the extension and also remove this common artifact from lib/mule too?